docs/iso-foreword.md

```diff
@@ -0,0 +1,46 @@
+# Foreword
+
+ISO (the International Organization for Standardization) and IEC (the
+International Electrotechnical Commission) form the specialized system for
+worldwide standardization. National bodies that are members of ISO or IEC
+participate in the development of International Standards through technical
+committees established by the respective organization to deal with particular
+fields of technical activity. ISO and IEC technical committees collaborate in
+fields of mutual interest. Other international organizations, governmental and
+non-governmental, in liaison with ISO and IEC, also take part in the work.
+
+The procedures used to develop this document and those intended for its further
+maintenance are described in the ISO/IEC Directives, Part 1. In particular, the
+different approval criteria needed for the different types of document should
+be noted (see
+[www.iso.org/directives](https://www.iso.org/directives) or
+[www.iec.ch/members_experts/refdocs](https://www.iec.ch/members_experts/refdocs)).
+
+Attention is drawn to the possibility that some of the elements of this
+document may be the subject of patent rights. ISO and IEC shall not be held
+responsible for identifying any or all such patent rights. Details of any
+patent rights identified during the development of the document will be in the
+Introduction and/or on the ISO list of patent declarations received
+(see [www.iso.org/patents](https://www.iso.org/patents)) or the IEC list of
+patent declarations received (see [patents.iec.ch](https://patents.iec.ch)).
+
+Any trade name used in this document is information given for the convenience
+of users and does not constitute an endorsement.
+
+For an explanation of the voluntary nature of standards, the meaning of ISO
+specific terms and expressions related to conformity assessment, as well as
+information about ISO's adherence to the World Trade Organization (WTO)
+principles in the Technical Barriers to Trade (TBT), see
+[www.iso.org/iso/foreword.html](https://www.iso.org/iso/foreword.html).
+In the IEC, see
+[www.iec.ch/understanding-standards](https://www.iec.ch/understanding-standards).
+
+This document was prepared by the Object Management Group (OMG) (as SPDX®
```

Comment:
Should we be mentioning both organizations here?

Reply:
@kestewart another one is The Linux Foundation?

Reply:
I would suggest using the following:

> This document was prepared by The Linux Foundation under the SPDX Working Group (as SPDX® Specification v3.0.1) and drafted in accordance with its editorial rules. Its preparation and publication has been made in coordination with related efforts with the Object Management Group (OMG).

@bobmartin3000 @kestewart Are you comfortable with this phrasing?

Reply:
@swinslow I have added your suggestion to the source, with additional "and its Contributors" which I take from the copyright line from #1010